From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@kunaldsingh__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kunalds). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Thraka commented:
@kunaldsingh you're doing this in the public repo instead of the private.

kunaldsingh commented:
@Thraka Yes, I thought we were going to move to public after Public Preview release. Is that not the case?

neilpeterson committed:
ingress doc

JasonWHowell committed:
Updating to new reference data experience

JasonWHowell committed:
Updating to new reference data experience

neilpeterson committed:
updates

JasonWHowell committed:
Updating to new reference data experience

JasonWHowell committed:
Updating to new reference data experience

JasonWHowell committed:
Updating to new reference data experience

JasonWHowell committed:
Updating to new reference data experience

JasonWHowell committed:
Updating to new reference data experience

JasonWHowell committed:
Updating to new reference data experience

JasonWHowell committed:
Updating to new reference data experience

JasonWHowell committed:
Updating to new reference data experience

JasonWHowell committed:
Updating to new reference data experience

JasonWHowell committed:
Updating to new reference data experience

JasonWHowell committed:
Updating to new reference data experience

JasonWHowell committed:
Updating to new reference data experience

mrbullwinkle committed:
.NET Core Draft 001

JasonWHowell committed:
Fixing screenshots

JasonWHowell committed:
Fixing screenshots

tdykstra committed:
webjobs sdk --> functions project

carolinacmoravia committed:
Revert "M22746: Should text "This link has broken (...)" get removed?"

carolinacmoravia committed:
Update azure-security-antimalware.md

mrbullwinkle committed:
.NET Core 002

neilpeterson committed:
updating to use Azure Vote helm chart for brevity

neilpeterson committed:
Azure samples helm repo

neilpeterson committed:
updated code

neilpeterson committed:
updates

neilpeterson committed:
copy

neilpeterson committed:
updated images

neilpeterson committed:
acrolinx

carolinacmoravia committed:
CC47892: Fix typo #437

Hello, @mumian,

This proposed file change comes from this Community Contribution: https://github.com/MicrosoftDocs/azure-docs.ja-jp/pull/437/files

Please, help to check this proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

neilpeterson committed:
updated with new helm chart

neilpeterson committed:
updated images

mrbullwinkle committed:
Draft 2_20_2018

mrbullwinkle committed:
small fixes'

MarkMcGeeAtAquent committed:
updates

mrbullwinkle committed:
Add intro + small fixes

MarkMcGeeAtAquent committed:
formatted heading, changed date

Commit:
Updating PKI content

MarkMcGeeAtAquent committed:
typos, duplicated text

Commit:
Updating PKI content

Commit:
updated PKI content based on feedback

Commit:
fixed small typo

Commit:
fixed small typo

neilpeterson committed:
updates

neilpeterson committed:
acrolinx

neilpeterson committed:
acrolinx

neilpeterson committed:
updates

neilpeterson committed:
updates

neilpeterson committed:
updates

mrbullwinkle committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into mrb_02_14_2018_dotnetcore

mrbullwinkle committed:
small changes'

mrbullwinkle committed:
Fix screenshot location

mrbullwinkle committed:
synthetic transaction

v-viinde committed:
USER STORY 1190927 SaaS App Tutorial: ZwayamSSO Update

MarkMcGeeAtAquent committed:
added alert to batches sample

MarkMcGeeAtAquent committed:
last fix

mrbullwinkle committed:
fix spacing, PowerShell

mrbullwinkle committed:
Fix head tag

MarkMcGeeAtAquent committed:
fixed include file

neilpeterson committed:
[AKS] adding acr authentication

neilpeterson committed:
typo

rwike77 committed:
updating metadata

rwike77 committed:
updated metadata

rwike77 committed:
Updating VM password

rwike77 committed:
acrolinx

TerryLanfear committed:
TOC update

Brenduns committed:
Identity for AzS Overview

Commit:
Updated PKI validation article

Commit:
Fixed a problem with Tenant level service plans confusing the removal script.

Commit:
Added a disclaimer about scripts being samples.

dlepow committed:
N-series: Windows Server 2016 RDMA support

neilpeterson committed:
updates / freshness

neilpeterson committed:
updates

MarkMcGeeAtAquent committed:
incorp feedback

Juliako committed:
Update TOC.md

sethmanheim committed:
Freshness reviews and updates

sethmanheim committed:
Acrolinx

mrbullwinkle committed:
incorp dev feedback

mrbullwinkle committed:
dev feedback

neilpeterson committed:
freshness / copy updates

neilpeterson committed:
Merge branch 'master' into aks-acr-auth-update

v-viinde committed:
Updated

Juliako committed:
Update postman-collection.md

v-viinde committed:
updated

Commit:
Adjust next steps.

Commit:
Removed vertical space.

mrbullwinkle committed:
dev feedback post ops build error

mrbullwinkle committed:
fix table alignment

Brenduns committed:
Fixing links

mrbullwinkle committed:
small fix

Brenduns committed:
Fixing links

mrbullwinkle committed:
fix link

Brenduns committed:
final links

mrbullwinkle committed:
more fixes

Commit:
Updates.

Commit:
Fixes.

mrbullwinkle committed:
Fix code block

mrbullwinkle committed:
code block issue

Commit:
Fix.

mattbriggs committed:
add space to next-line works and auth script works

mattbriggs committed:
update other occurences

Commit:
Fixed Warning block

Commit:
Updated for acrolinx results

mimig1 committed:
Cosmos DB: Deprecating run-hadoop-with-hdinsight article and redirecting

tdykstra committed:
prevent right nav entries from wrapping

Brenduns committed:
fixing issues found in PR

mimig1 committed:
Cosmos DB: Deprecating some explorer panes

Brenduns committed:
trying to get first instance of Italic and < to render. Later instances that are identicial render fine...

mimig1 committed:
Fixing validation error

tdykstra committed:
fix indentation and headings

GitHubber17 committed:
Merge pull request #33644 from mrbullwinkle/mrb_02_14_2018_dotnetcore

dotnetcore app insights doc updates

Brenduns committed:
Rendering system wins. Can't make it work. DOn't have time. Change of display structure.

Lisaco88 committed:
Removed bold where needed

v-geberr committed:
edits

anzaman committed:
Update vpn-gateway-certificates-point-to-site.md

added the -DnsName parameter for StrongSwan

Commit:
Grammar fixes from Acrolinx

PRMerger18 committed:
Merge pull request #33546 from carolinacmoravia/revert-32852-patch-601

Revert "M22746: Should text "This link has broken (...)" get removed?"

Commit:
More Acrolinx fixes.

Commit:
Merge pull request #33916 from mattbriggs/mdb_20180223_scripttypo

script typo

KaitlynCorazao committed:
Update advisor-overview.md

Commit:
Merge branch 'master' into GBLPowerShellRemovalFix

Lisaco88 committed:
Merge pull request #33645 from MarkMcGeeAtAquent/table-update-php

Cosmos DB: update Storage Table API for PHP article

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into new-acs

Lisaco88 committed:
Merge pull request #33837 from neilpeterson/aks-acr-auth-update

[AKS] adding acr authentication

PRMerger13 committed:
Merge pull request #33857 from dlepow/fix2

N-series: Windows Server 2016 RDMA support

Commit:
Merge pull request #33918 from v-geberr/0223-appInsights

edits

Lisaco88 committed:
Merge pull request #33897 from jimdial/patch-76

Adjust next steps.

PRMerger18 committed:
Merge pull request #33915 from piotrci/GBLPowerShellRemovalFix

Gbl power shell removal fix - try 2

KaitlynCorazao committed:
Update advisor-get-started.md

v-shils committed:
Merge pull request #33852 from Brenduns/identity-docs

Identity for AzS Overview

tamram committed:
updates for MVC guidance

samedder committed:
Transfering SF CLI related articles to new owner

tamram committed:
updates for MVC guidance

Commit:
Merge pull request #33868 from sethmanheim/wk-2-22

Freshness reviews and updates

rwike77 committed:
removed article

PRMerger14 committed:
Merge pull request #33919 from kasparks/patch-1

Update advisor-overview.md snooze to postpone

PRMerger19 committed:
Merge pull request #33846 from rwike77/patch-29

Updating VM password

tdykstra committed:
fix links

Commit:
Merge pull request #33920 from kasparks/patch-2

Update advisor-get-started.md snooze to postpone

mimig1 committed:
Cosmos DB: Updating logging image.

rwike77 committed:
Updated examples

mimig1 committed:
additional changes.

PRMerger14 committed:
Merge pull request #33921 from samedder/transfer_cli_articles

Transfering SF CLI related articles to new owner

Lisaco88 committed:
Merge pull request #33912 from mimig1/02232018

Cosmos DB: Deprecating run-hadoop-with-hdinsight article and redirecting

PRMerger12 committed:
Merge pull request #33606 from carolinacmoravia/patch-619

CC47892: Fix typo #437

rwike77 committed:
fixed typo

mimig1 committed:
Cosmos DB: Added new reader role to AD docs

PRMerger17 committed:
Merge pull request #33924 from rwike77/patch-30

Updated examples

zhiweiwangmsft committed:
reorder the format

reorder the format

v-shils committed:
Merge pull request #33923 from mimig1/022320183

Cosmos DB: Updating logging image

v-shils committed:
Merge pull request #33849 from TerryLanfear/new-acs

TOC update

Commit:
Merge pull request #33926 from zhiweiwangmsft/patch-10

reorder the format

PRMerger16 committed:
Merge pull request #33917 from anzaman/patch-32

Update vpn-gateway-certificates-point-to-site.md

Lisaco88 committed:
Merge pull request #33580 from tdykstra/queues

update functions docs from webjobs sdk docs

mimig1 committed:
Adding new role to the top table of roles

tedway committed:
Updated ms.date

JasonWHowell committed:
Update time-series-insights-add-reference-data-set.md

Lisaco88 committed:
Merge pull request #33922 from tamram/tamram-0223

qs updates for MVC guidance

HeidiSteen committed:
R Server for HDInsight - fixing links that resolve to machine learning server content pages

DhruvMsft committed:
Limitations updated with Web App limitation

Commit:
Merge pull request #33927 from tedway/patch-5

Updated ms.date

v-shils committed:
Merge pull request #33840 from rwike77/metadata

updating metadata

mimig1 committed:
Corrected alphabetical order of new items in doc

Commit:
freshness and fixing links

v-shils committed:
Merge pull request #33445 from JasonWHowell/tsireferencedata

TSI Revising reference data

PRMerger12 committed:
Merge pull request #33929 from HeidiSteen/heidist-master

R Server for HDInsight - fixing links that resolve to machine learnin…

PRMerger13 committed:
Merge pull request #33928 from DhruvMsft/patch-1

Limitations updated with Web App limitation

Commit:
fixing link

PRMerger14 committed:
Merge pull request #33925 from mimig1/022320184

Cosmos DB: Added new reader role to AD docs

Commit:
Merge pull request #33931 from Blackmist/freshness

freshness and fixing links

mimig1 committed:
Acrolinx fixes

markgalioto committed:
updating metadata for a few Backup articles pertaining to monitoring

GitHubber17 committed:
Merge pull request #33862 from Juliako/patch-1

Update TOC.md

rmca14 committed:
Merge pull request #33659 from jeffgilb/jeff222

Updating PKI content

GitHubber17 committed:
Merge pull request #33913 from mimig1/022320182

Cosmos DB: Deprecating some explorer panes

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger10 committed:
Merge pull request #33937 from markgalioto/ab-2-23-2018

updating metadata for a few Backup articles pertaining to monitoring

rmca14 committed:
Merge pull request #33430 from neilpeterson/aks-ingress-tls

[AKS] ingress and TLS termination

mahesh-unnikrishnan committed:
Update active-directory-ds-admin-guide-configure-secure-ldap-enable-ldaps.md

v-alje committed:
Merge pull request #33941 from MicrosoftDocs/FromPublicRepo

From public repo

MarkMcGeeAtAquent committed:
overlooked

rjagiewich committed:
Merge pull request #33799 from v-viinde/zwayamssoupdate

USER STORY 1190927 SaaS App Tutorial: ZwayamSSO Update

PRMerger15 committed:
Merge pull request #33942 from mahesh-unnikrishnan/patch-17

Update active-directory-ds-admin-guide-configure-secure-ldap-enable-l…

Commit:
Merge pull request #33944 from MarkMcGeeAtAquent/quick-fix

Cosmos DB: fix for overlooked feedback comment
